performance-tests/test-app/element-rendering.js

```diff
@@ -20,7 +20,8 @@ module.exports = () => {
 	}
 
 	const numberOfElements = Number.parseInt(pageStore.elements, 10)
-	const elements = [...new Array(Number.isNaN(numberOfElements) ? 0 : numberOfElements)].map(() => html`<span>-</span>`)
+	const newSpan = () => html`<span>-</span>`
+	const elements = [...new Array(Number.isNaN(numberOfElements) ? 0 : numberOfElements)].map(newSpan)
```

Comment:
This is only because it makes it easier to inspect on the performance tab (otherwise it just appears as `anonymous function`